control-users/src/main/resources/META-INF/resources/spring-map-jsp/register.jsp

```diff
@@ -298,6 +295,10 @@
                             errorMsg("#email", '<spring:message code="user.registration.error.emailExists"/>');
                         } else if (errorResponse.toLowerCase().indexOf("username") >= 0) {
                             errorMsg("#username", '<spring:message code="user.registration.error.usernameExists"/>');
+                        } else if (errorResponse.toLowerCase().indexOf("address") >= 0) {
+                            errorMsg("#email", '<spring:message code="user.registration.error.sendingFailed"/>');
```

Comment:
Add js escaping for all of these:  <spring:message javaScriptEscape="true" .../>